Dockerfile

```diff
@@ -0,0 +1,109 @@
+# syntax=docker/dockerfile:1
+
+# Declare build arguments
+# TODO: this is only used for node image right now, should we also use it for nodeplugin?
```

Comment:
Yeah we should use it for all

Reply:
just checked, nodeplugin doesn't even define those variables. Let's do it in a separate PR if it's ever needed. Probably add it to all binaries in one go.